### ISSUE TITLE
[Accessibility][Keyboard navigation]: User have to re-navigate the checkbox options to uncheck/check them at the same time.

### ISSUE THREAD

**Issue by joschect** [OfficeDev/office-ui-fabric-react#11198]
Split out of issue #6920

**Comment by xugao** [comment]
agree this is a bug. user should be able to unselect after select without navigation
